Assets/Content/World/Ambience/AmbienceHandler.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections;
+using UnityEngine;
+
+public class AmbienceHandler : MonoBehaviour
+{
+    
+    [Header("Current Status")]
+    
+    //TODO: Change this based on how much air is in the room.
+    [Tooltip("How much air is in the room. This effects the volume of the audio sources.")]
+    [Range(0f,1f)]public float air = 1;
+    //TODO: Change this based on air flow in the room.
+    [Tooltip("How windy the air is in the area. This effects how the air sounds.")]
+    [Range(0f, 1f)] public float windiness;
+    //TODO: Change this based on area's power.
+    [Tooltip("The amount of power in the area. Lower levels means quieter and deeper electrical humming.")]
+    [Range(0f, 1f)]public float power = 1;
+    [Tooltip("What clip is currently being played by the ambient noise generator. You don't need to mess with this, just for debugging purposes.")] 
+    [SerializeField]private AudioClip nowPlaying;
```

Comment:
Please  make sure that there's a space between the attribute and the access modifier.

Reply:
done!